tests/acceptance/features/apiOcm/share.feature

```diff
@@ -288,3 +288,139 @@ Feature: an user shares resources usin ScienceMesh application
       | Space Viewer     |
       | Space Editor     |
       | Manager          |
+
+  @issue-9908
+  Scenario: sharer lists the shares shared by him to a federated user
+    Given using server "LOCAL"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+    And user "Alice" has uploaded file with content "ocm test" to "/textfile.txt"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | textfile.txt                  |
+      | space           | Personal                      |
+      | sharee          | Brian                         |
+      | shareType       | user                          |
+      | permissionsRole | Viewer                        |
+      | federatedServer | @federation-ocis-server:10200 |
+    When using server "LOCAL"
+    And user "Alice" lists the shares shared by him using the Graph API
```

Comment:
We are already in `LOCAL` server so I think  we can skip `When using server "LOCAL"`
```suggestion
    When user "Alice" lists the shares shared by him using the Graph API
```